src/BM2/DungeonBundle/DataFixtures/ORM/LoadDungeonCards.php

```diff
@@ -45,10 +47,10 @@ class LoadDungeonCards extends AbstractFixture implements OrderedFixtureInterfac
 		'action.heal3'		=> array('rarity' =>  100, 'monsterclass'=> '', 'target'=>array('monster'=>false, 'treasure'=>false, 'dungeoneer'=>true)),
 		'action.rest3'		=> array('rarity' =>   75, 'monsterclass'=> '', 'target'=>array('monster'=>false, 'treasure'=>false, 'dungeoneer'=>false)),
 		'action.untrap3'	=> array('rarity' =>  100, 'monsterclass'=> '', 'target'=>array('monster'=>false, 'treasure'=>true, 'dungeoneer'=>false)),
-		'scout.tripple'	=> array('rarity' =>   50, 'monsterclass'=> '', 'target'=>array('monster'=>false, 'treasure'=>false, 'dungeoneer'=>false)),
+		'scout.tripple'		=> array('rarity' =>   50, 'monsterclass'=> '', 'target'=>array('monster'=>false, 'treasure'=>false, 'dungeoneer'=>false)),
 		'fight.kill'		=> array('rarity' =>  100, 'monsterclass'=> '', 'target'=>array('monster'=>true, 'treasure'=>false, 'dungeoneer'=>false)),
 		'fight.sure'		=> array('rarity' =>  100, 'monsterclass'=> '', 'target'=>array('monster'=>true, 'treasure'=>false, 'dungeoneer'=>false)),
-		'fight.tripple'	=> array('rarity' =>  100, 'monsterclass'=> '', 'target'=>array('monster'=>true, 'treasure'=>false, 'dungeoneer'=>false)),
+		'fight.tripple'		=> array('rarity' =>  100, 'monsterclass'=> '', 'target'=>array('monster'=>true, 'treasure'=>false, 'dungeoneer'=>false)),
```

Comment:
please keep indent on tabs. Otherwise you will have these changes in the diff which actually are no changes. It just makes it harder to understand what was actually changed.